charts/dbildungs-iam-server/config/config.json

```diff
@@ -45,5 +45,11 @@
     "KEYCLOAK_ADMINISTRATION_MODULE_LOG_LEVEL": "debug",
     "HEALTH_MODULE_LOG_LEVEL": "debug",
     "BACKEND_FOR_FRONTEND_MODULE_LOG_LEVEL": "debug"
+  },
+  "ITSLEARNING": {
+    "ENABLED": false,
+    "ENDPOINT": "https://localhost",
+    "USERNAME": "username",
```

Comment:
hmmm, looks a little weird as a default, maybe https://itslearning.example.com would be better to show that it is indeed **not** a local service